### PR TITLE
Adding BO page to landing page

### PR DESCRIPTION
**Pull request by javiergonzalezh**
*Issue #, if available:*

*Description of changes:* I have added a blog including code to provide an introduction to BO with emukit.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: _posts/2016-06-12-bayesian-optimization.markdown [hunk 1]
```diff
@@ -6,4 +6,103 @@ img: bayesian_optimization.jpeg
...
+from emukit.core.loop import FixedIterationsStoppingCondition
+
+stopping_condition = FixedIterationsStoppingCondition(i_max = 20)
+bayesopt_loop.run_loop(f, stopping_condition)
+```
+And that's it! You can check the obtained the result looking into the state of the loop. Note that you can use other models, including those with multiple outputs,
+and acquisitions of your own in this loop.
+
+We’re always open to contributions! Please read our [contribution guidelines](CONTRIBUTING.md) for more information. We are particularly interested in contributions
+regarding translations and tutorials.
```
Comment (apaleyes) [inline _posts/2016-06-12-bayesian-optimization.markdown]: translations?

File: _site/documentation/index.html [hunk 2]
```diff
@@ -1 +1 @@
...
...ntainer"> <div class="row p20"> <div class="col-md-4 text-center mt25"> <a target="_blank" href="http://twitter.com/@emukitUQ"><li class="social twitter"><i class="fa fa-twitter-square"></i></li></a> <a target="_blank" href="https://github.com/amzn/emukit"><li class="social github"><i class="fa fa-github-square"></i></li></a> <a target="_blank" href="mailto:emukit-mailist@amazon.com"><li class="social email"><i class="fa fa-envelope"></i></li></a> </div> </div> </div> </footer> <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script> <script src="/emukit/js/bootstrap.min.js"></script> <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> <!-- Google Analytics Tracking code --> <script> (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,'script','//www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-83979019-1', 'auto'); ga('send', 'pageview'); </script> </body> </html>
```
Comment (apaleyes) [inline _site/documentation/index.html]: Any idea why these files get update on each and every PR?

File: _posts/2016-06-12-bayesian-optimization.markdown (hunk 1 above)
Comment (javiergonzalezh) [reply _posts/2016-06-12-bayesian-optimization.markdown]: examples :)

File: _site/documentation/index.html (hunk 2 above)
Comment (javiergonzalezh) [reply _site/documentation/index.html]: No idea, I guess it has to do with the way the web is regenerated every time a change is introduced

File: _posts/2016-06-12-bayesian-optimization.markdown [hunk 3]
```diff
@@ -6,4 +6,103 @@ img: bayesian_optimization.jpeg
...
+The bach size is set to one in this example as we'll collect evaluations sequentially but parallel evaluations are allowed. Once the loop is created we run it for some iterations,
+20 in our example.
+
+```
+from emukit.core.loop import FixedIterationsStoppingCondition
+
+stopping_condition = FixedIterationsStoppingCondition(i_max = 20)
+bayesopt_loop.run_loop(f, stopping_condition)
+```
+And that's it! You can check the obtained the result looking into the state of the loop. Note that you can use other models, including those with multiple outputs,
```
[the remark this answers is not in the document]
Comment (javiergonzalezh) [reply _posts/2016-06-12-bayesian-optimization.markdown]: I'll remove the multi-output for the moment. We can add it once we have an example.

File: _posts/2016-06-12-bayesian-optimization.markdown [hunk 4]
```diff
@@ -6,4 +6,103 @@ img: bayesian_optimization.jpeg
...
+
+```python
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement
+
+expected_improvement = ExpectedImprovement(model = model_emukit)
+```
+
+Given these ingredients, Bayesian optimization iterates the following three steps until it achieves a predefined stopping criterion.
+
+1. Find the next point to evaluate $$x_{n+1}$$ by using a numerical solver maximize $$a(x)$$.
```
[the remark this answers is not in the document]
Comment (javiergonzalezh) [reply _posts/2016-06-12-bayesian-optimization.markdown]: done

**Comment by javiergonzalezh** [comment]
I have removed all the maths from the notebook.

**Pull request merged**